Dockerfile

```diff
@@ -9,6 +9,4 @@ ADD ./app ./app
 ADD ./main.sh ./main.sh
 
 # set up entrypoint
-CMD ["./main.sh"]
-ARG PORT=8080
-EXPOSE $PORT
\ No newline at end of file
+ENTRYPOINT ["./main.sh"]
```

Comment:
this is what happens in windows if you dont explicitly use bash on the ENTRYPOINT in the first element when starting the container.

/usr/bin/env: â€˜bash\râ€™: No such file or directory

Reply:
and the same thing happens in k8s.

Reply:
I've managed to deploy this with minikube on MacOS without issues. I also can't find any indication on the internet of others having this issue with bash scripts and k8s, so I have to suspect that something else is going on in your particular case. Can you provide more details about what you're trying: host OS, docker/k8s versions, etc.?

Reply:
windows 10 and docker desktop (both latest versions) integrated with PyCharm. 

i will see what happens when i create the image on WSL.

Reply:
it appears to be a dos vs. linux line endings issue. WSL reports: "/usr/bin/env: ‘bash\r’: No such file or directory"

Reply:
converting main.sh to use unix line endings fixed the issue